src/common/wallet/cointype.js

```diff
@@ -62,53 +69,60 @@ const coinType = {
     symbol: 'BTC',
   },
   RBTCTestnet: {
-    networkId: 37310,
+    networkId: 31,
+    pathNetworkId: 37310,
     icon: references.images.RBTC_GREY,
     defaultName: 'Smart Bitcoin',
     chain: 'Rootstock',
     type: 'Testnet',
     symbol: 'RBTC',
   },
   RIFTestnet: {
-    networkId: 37310,
+    networkId: 31,
+    pathNetworkId: 37310,
     icon: references.images.RIF_GREY,
     defaultName: 'RSK Infra Framework',
     chain: 'Rootstock',
     type: 'Testnet',
     symbol: 'RIF',
   },
   DOCTestnet: {
-    networkId: 37310,
+    networkId: 31,
+    pathNetworkId: 37310,
     icon: references.images.DOC_GREY,
     defaultName: 'Dollar on Chain',
     chain: 'Rootstock',
     type: 'Testnet',
     symbol: 'DOC',
   },
   RIFPTestnet: {
-    networkId: 137,
+    networkId: 31,
+    pathNetworkId: 137,
```

Comment:
If it is Testnet, the pathNetworkId is 37310, like the others, isn't it?

Reply:
Yes, you are right! 👍